harvard-lts/CURIOSity
# Push SoftServ 1-04-2023

**Push SoftServ 1-04-2023**
* * *

# What does this Pull Request do?
Pushes changes from SoftServ 1-04-2023 including:

- revert to rails 6.1.5
- lock at spotlight version 3.3.0
- link facet searches
- Fix download issue

# How should this be tested?

Ideally test locally in prod mode, make sure you can access exhibits and follow the testing instructions for facet linking (see: https://github.com/scientist-softserv/harvard-curiosity/pull/29) and try a download as seen in this pr (https://github.com/scientist-softserv/harvard-curiosity/pull/28)


# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
